Bottom: 8339e8b074b7cec0a615f3a288826d91c489625e
Top:    b18ff0542719b3c4900b3df9454ecb64b180e252
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-07 16:13:52 +0100

Refresh of make-rats-snap-to-the-center-o

---

diff --git a/src/find.c b/src/find.c
index 514cfdd..edf2f7e 100644
--- a/src/find.c
+++ b/src/find.c
@@ -1372,7 +1372,6 @@ ArcArcIntersect (ArcTypePtr Arc1, ArcTypePtr Arc2)
 static Boolean
 IsRatPointOnLine (PointTypePtr Point, LineTypePtr Line)
 {
-  printf ("IsRatPointOnLine (%i,%i)\n", Point->X, Point->Y);
   if ((Point->X == Line->Point1.X && Point->Y == Line->Point1.Y) ||
       (Point->X == Line->Point2.X && Point->Y == Line->Point2.Y) ||
       (Point->X == (Line->Point1.X + Line->Point2.X) / 2 &&
